pkgs/by-name/gi/git-up/package.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  fetchPypi,
+  python3Packages,
+  writableTmpDirAsHomeHook,
+  gitMinimal,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "git-up";
+  version = "2.3.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    pname = "git_up";
+    inherit version;
+    hash = "sha256-SncbnK6LxsleKRa/sSCm/8dsgPw/XJGvYfkcIeWYDy4=";
+  };
+
+  pythonRelaxDeps = [
+    "termcolor"
+  ];
+
+  build-system = with python3Packages; [
+    poetry-core
+  ];
+
+  # required in PATH for tool to work
+  propagatedBuildInputs = [ gitMinimal ];
```

Comment:
I don't think this is the right place for this. Perhaps @GaetanLepage could confirm this ?

Reply:
wrapPythonPrograms in pkgs/development/interpreters/python/wrap.sh wraps the $out/bin entrypoints in `bin` with the propagated build inputs